### PR TITLE
refactor(main page) - update structure + changed contentful service

### PR DESCRIPTION
**Pull request by dydek**

### CODE REVIEW

File: app/shared/services/contentful-content.service.ts [hunk 1]
```diff
@@ -92,6 +103,27 @@ export class ContenfulContent {
       .commit();
   }
 
+  private getLatestItems<T>(request: ContentfulRequest, limit: number
```
Comment (rychkog) [inline app/shared/services/contentful-content.service.ts:90]: Michal, I am wondering - what is the way of specifying order direction?

File: app/shared/services/contentful-content.service.ts [hunk 2]
```diff
@@ -66,11 +77,11 @@ export class ContenfulContent {
    * @param slug
    * @returns {any}
    */
-  getNodePage(slug: string): Observable<NodePageContent> {
+  getNodePage(slug: string): Observable < NodePageContent > {
```
Comment (rychkog) [inline app/shared/services/contentful-content.service.ts]: In some places you have spaces between angle brackets in some - don't like in the code below
`transformResponse<ContentfulNodePage>`

Let's stick to convention having them like 

`transformResponse<ContentfulNodePage>`

Comment (dydek) [inline app/shared/services/contentful-content.service.ts]: ok, sounds good to me, this one is from pycharm auto formatting - you have right ... 
ok, now I know, when I was refactoring the code, I'd removed one bracket and then reformatted the code with pycharm, this is the reason why it looks strange. So it's my mistake, not my intent to have different formatting style.

File: app/shared/services/contentful-content.service.ts (hunk 1 above)
Comment (dydek) [inline app/shared/services/contentful-content.service.ts:90]: `latestItems` are sorted by `sys.createdAt` by default, but if we'd need some custom sorting there is an `order` method for sorting records.

**Comment by rychkog** [comment]
:+1:

**Pull request merged**